src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    email,
+    name,
+  } = user;
```

Comment:
```suggestion
  const { email, name } = user;
```
You can leave it in one line, if there are < 3 props